km_api/know_me/models.py

```diff
@@ -283,6 +203,117 @@ def __str__(self):
         return self.user.get_short_name()
 
 
+class ListContent(models.Model):
+    """
+    Model to store list content for a :class:`.ProfileItem`.
+    """
+    profile_item = models.OneToOneField(
+        'know_me.ProfileItem',
+        on_delete=models.CASCADE,
+        related_name='list_content',
+        verbose_name=_('profile item'))
+    """
+    :class:`.ProfileItem`:
+        The profile item that the list content is attached to.
+    """
+
+    class Meta(object):
+        verbose_name = _('profile item list content')
+        verbose_name_plural = _('profile item list content')
```

Comment:
Should the verbose name plural be plural or is there no plural for list content?